concepts/05 UI Components/Autocomplete/00 Getting Started with Autocomplete/25 Customize Item Appearance.md

```diff
@@ -0,0 +1,91 @@
+To customize appearance of all Autocomplete suggestions, define an [itemTemplate](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#itemTemplate). To customize an individual suggestion, specify the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property of the item. Note that Angular and Vue use [custom templates](/Documentation/Guide/UI_Components/Common/Templates/#Custom_Templates) instead of the [template](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#template) property. In React, specify the [render](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#render) or [component](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/items/#component) property.
+
+If your data is grouped, you may want to customize the group headings. Define a [groupTemplate](/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#groupTemplate) for that purpose.
```

Comment:
as far as you mention react specific, you can also point out here, that for React `itemRender` and `groupRender` should be used